docs/Makefile

```diff
@@ -27,9 +27,13 @@ BUILDDIR      = build
 help:
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
-.PHONY: help Makefile
+create_rst_examples:
+	@echo "Creating rST files for SYCL* Examples..."
+	py -3 rst_examples.py
```

Comment:
You introduced dependency to 'py' here. Please do it carefully with appropriate diagnostics for users, if she have no correct environment. Also, introduce appropriate variable to make it possible to specify full path via command line argument for users.

Reply:
Users are not expected to use this document generation scripts